docs/CONTRIBUTING.md

```diff
@@ -9,6 +9,27 @@ repository. To propose improvements or fix a bug, feel free to submit a PR.
 
 Before we can accept your contributions you have to sign the [Contributor License Agreement][0]
 
+## Pull Requests
```

Comment:
Unrelated to this PR, but I thought this paragraph should be upper in the page as it's more human-related than technical.